src/common/hostname.h

```diff
@@ -14,9 +14,9 @@
 
 #ifndef CEPH_COMMON_HOSTNAME_H
 #define CEPH_COMMON_HOSTNAME_H
-
 #include <string>
+#include <unistd.h>
```

Comment:
please move this `#include` into `hostname.cc`.

Reply:
@tchaikov make sense